cdm/src/main/java/ucar/nc2/constants/FeatureType.java

```diff
@@ -49,6 +50,7 @@
   FMRC,       // two time dimensions, runtime and forecast time
   SWATH,      // 2D latlon, dependent time, polar orbiting satellites
   CURVILINEAR,// 2D latlon, independent time
+  SIMPGEO,	  // new simple geometry coverage type

```

Comment:
With the exception of FMRC and UGRID, most constants in this file spell out words in their entirety. Should this enum entry be exceptional as well? If so, why?

Reply:
Perhaps because SIMPLEGEOMETRY would be monstrously long? Though if that is not the case it wouldn't be too difficult to change it to SIMPLEGEOMETRY.

Reply:
Alright. Changing it to SIMPLE_GOMETRY.